src/components/cc-addon-option-form/cc-addon-option-form.stories.js

```diff
@@ -12,7 +12,7 @@ const htmlExample = html`
     Donec rutrum ante vel dolor bibendum, eu pretium velit gravida
   </div>
   <div class="option-warning">
-    <cc-icon .icon="${iconAlert}" accessible-name="${'cc-tile-deployments.error.icon-a11y-name'}" class="icon-warning"></cc-icon>
+    <cc-icon .icon="${iconAlert}" accessible-name="Warning" class="icon-warning"></cc-icon>
```

Comment:
Why did we go for a translated message to a hard coded english one?

Reply:
Well, it was a story and I found it overkill to add a translation just for this. But I can if you want, just tell me.

Reply:
NP